modules/market/src/main/java/com/opengamma/strata/market/observable/QuoteId.java

```diff
@@ -123,6 +124,19 @@ public QuoteId withObservableSource(ObservableSource obsSource) {
     return new QuoteId(standardId, fieldName, obsSource);
   }
 
+  /**
+   * Returns a security identifier based on the standard identifier.
+   * <p>
+   * This convenience method allows a quote identifier to be converted to a security identifier.
+   * Since not all quotes are for securities, there is no guarantee that a security exists
+   * for the resulting identifier.
+   * 
+   * @return a security identifier formed by wrapping the standard identifier of this instance
+   */
+  public SecurityId toSecurityId() {
```

Comment:
Is this method worth having given how simple it is and how a quote isn't guaranteed to be for a security?

Would it be better to have `SecurityId.of(QuoteId)` so you're putting the onus on the user of knowing a quote represents a security?